scripts/test_screeps_tencent_batch_rl_runner.py

```
@@ -0,0 +1,359 @@
+                            "StartTime": "2026-05-17T00:00:00Z",
+                        },
+                        {
+                            "ActivityId": "new",
+                            "ActivityType": "SCALE_OUT",
+                            "StatusCode": "FAILED",
+                            "StartTime": "2026-05-17T00:01:00Z",
+                        },
+                    ]
+                }
+
+        controller = FakeController(args=controller_args(), run_id="run-test", artifact_dir=Path("/tmp/run-test"))
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/run-test"

(S108)